pkg/apis/componentconfig/types.go

```diff
@@ -772,6 +773,14 @@ type KubeControllerManagerConfiguration struct {
 	// Zone is treated as unhealthy in nodeEvictionRate and secondaryNodeEvictionRate when at least
 	// unhealthyZoneThreshold (no less than 3) of Nodes in the zone are NotReady
 	UnhealthyZoneThreshold float32
+	// Reconciler runs a periodic loop to reconcile the desired state of the with
+	// the actual state of the world by triggering attach detach operations.
+	// This flag enables or disables reconcile.  Is false by default, and thus enabled.
+	DisableReconciliation bool
+	// ReconcilerSyncDuration is the amount of time the reconciler sync states loop
+	// wait between successive executions. Is set to 5 min by default.
```

Comment:
These names should make it clearer these options are related to volume attach/detach reconciliation